src/components/mediaLibraryCreator/mediaLibraryCreator.js

```diff
@@ -52,9 +52,13 @@ define(['loading', 'dialogHelper', 'dom', 'jQuery', 'components/libraryoptionsed
     }
 
     function initEditor(page, collectionTypeOptions) {
-        $('#selectCollectionType', page).html(getCollectionTypeOptionsHtml(collectionTypeOptions)).val('').on('change', function () {
+        const selectCollectionType = $('#selectCollectionType', page);
```

Comment:
Why this one it's still using JQ?
```suggestion
        const selectCollectionType = page.querySelector('#selectCollectionType');
```